scss/chart.scss

```diff
@@ -1050,7 +1050,7 @@ body {
         }
     }
 
-    .map-polyline{
+    .map-polyline g {
```

Comment:
This was overriding inline styles for the `<path..` so I made this more specific so that inline styles actually work when passed through `popupStyle` getter/setter